src/main/java/com/epam/aidial/cfg/service/transfer/importer/RoleImporter.java

```diff
@@ -43,6 +43,7 @@ public Collection<ImportComponent<Role>> importAdminRoles(Map<String, Role> role
                     .map(roleEntry -> {
                                 var role = roleEntry.getValue();
                                 role.setName(roleEntry.getKey());
+                                role.setDisplayName(roleEntry.getKey());
```

Comment:
roleEntry.getKey() used in 3 lines in a row. put it in var pls

Reply:
done